contact/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Contact Us
+image:
+    heading: mdst-logo.png
+share: false
+comments: false
+
+---
+
+We use a Google Form to handle our contact page, so just click the link below and you'll be taken there.
+
+[Contact Form Found Here!](https://docs.google.com/forms/d/1Pi_KLWh_7I3FeJTVyOWcYOwN02BaT7Ke843VOPG-0ZI/viewform)
+
+
+<!--
+This is an embedded version of the form at the link above. In case anyone wants to switch
+to the ebedded version, they can do so by deleting this comment.
+
+<iframe src="https://docs.google.com/forms/d/1Pi_KLWh_7I3FeJTVyOWcYOwN02BaT7Ke843VOPG-0ZI/viewform?embedded=true" width="640" height="1000" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
+-->
```

Comment:
On second thought, we really shouldn't leave comments in the website. We should commit now. Y'all want the form embedded, or just a link to it? I like using a link because if someone is visiting on mobile, the page will load faster, and they can easily navigate elsewhere to the site. Also, the form displays nicely on mobile only if it's loaded as a separate page.